app/build.gradle

```diff
@@ -10,6 +10,7 @@ plugins {
 android {
     compileSdk 31
     buildToolsVersion '31.0.0'
```

Comment:
I think this line can be removed. If we just keep the Gradle Plugin updated.

https://developer.android.com/studio/releases/build-tools

"If you're using [Android plugin for Gradle 3.0.0](https://developer.android.com/studio/releases/gradle-plugin#3-0-0) or higher, your project automatically uses a default version of the build tools that the plugin specifies."

This way we make sure that the Plugin uses the recommended/tested/default version.

Reply:
We should nevertheless specify it to ensure the usage of the same version when building NewPipe by hand. Otherwise, F-Droid might make problems resulting in non-reproducible builds.